### DISCUSSION TITLE
My account is not authorized

### DISCUSSION THREAD

**Discussion by itsmehul** [community/community#19466] [Copilot Conversations] [answered]
I have had the preview for a while now. I went in to add my payment details but after it was added github said my account is blocked for payments, next they are willing to check in July 11. I tried again this time skipping the payment option yet no changes have reflected on my copilot service. It keeps asking me to first sign-up, upon realising I have it, it directs me to payment option, I have tried all options but I'm still stuck in this loop. One thing I suspect is my account being blocked for payments. I tried updating my details but they will confirm much later, when i want them to verify it asap.

**Comment by melvincwng** [comment, marked as answer]
Hi @itsmehul, please raise a ticket to [GitHub Support](https://support.github.com/contact/account) and allow them a couple of working days to investigate and resolve the issue for you.

In the meanwhile, please kindly wait patiently for their response.

Hope this helps.

**Reply by melvincwng** [reply]
@itsmehul, if this reply is helpful, please 'Mark as answer' so that we know this issue has been addressed.

Thank you and have a nice day :)
